Gaurav.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
+
+
+public class Gaurav {
+
+//	 private static int count = 0;
+
+	
+	public static void main(String[] args) {
+		
+				System.out.println("This is Gaurav Garg testing using GIT");
+				
+				Process process =null;
+		
+			//System.out.println("This is Gaurav Garg testing using GIT");		
+				try {
+					process.getOutputStream().close();
+				} catch (IOException e) {
+					// TODO Auto-generated catch block
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1135)